test/e2e/pages/allocation-criteria.js

```diff
@@ -72,6 +87,57 @@ class AllocationCriteriaPage extends Page {
       }
     }
 
+    switch (estateAnswer) {
```

Comment:
An alternative could be to call `fillInForm` twice. The first call will return you the random value that was selected and then you won't have to maintain the list of options within the test file too.

Reply:
That was a good idea. Amended.